_posts/2021-06-08-product-management-in-open-source.md

```diff
@@ -0,0 +1,116 @@
+
+
+## From open source project to product
+
+An archetypal open source project begins with the development of an initial solution and its deployment by the originators. The solution is created to solve a specific problem within the organization. But as we know, many problems are not unique to a particular organization.
+
+The code is made public and the project is completed. A few months later, another organization learns about the solution and would like to use it. But the solution is tailored for use within a specific organization. For example, the house style may not be adaptable, the business logic may be different or not configurable, and authentication and authorization may be handled differently. User documentation is also sometimes missing. You want to use it but wonder "Who is actually responsible for this product? And how do I ensure that my requirements and wishes find their way into the software solution?"
+
+When you buy a commercial product, this product responsibility is paid off. You trade control and freedom for convenience.
+
+At a products company, there are people working every day to ensure that a software solution can be used by the largest possible group of users. Research is conducted into the needs, requirements and desires of users. These requirements and desires are identified, prioritized and finally developed. A lot of attention is also paid to usability, documentation and training. Information is made available in different languages and user research is carried out regularly. This type of work is also called 'product management'.
+
+Thus, a lot of work and effort is required before a project becomes a product. As more users adopt the product, elements in the project will need to be made generic and configurable.
+
+In open source software, those responsibilities and activities are a shared task of the users. By users we mean not only end users but also developers from both public and private organizations. Together, these users will have to invest time and money. The more people use the product and invest in it, the more you as a user can benefit from each other's efforts. As a user you are therefore responsible for product management. This requires commitment, knowledge and perseverance.
+
+
+## What is product management?
+
+[Product management ](https://en.wikipedia.org/wiki/Product_management)is a collection of product development and product marketing activities within a products or services organization. The purpose of these activities is to develop a product that meets market demand on the one hand and to manage the product throughout its life cycle on the other. At a large company like Apple, thousands of people work on this every day. The end goal is to make a product or service that matches the market so well that as many as possible can be sold. The activities and responsibilities are centralized because only one organization develops the product and determines what happens to it.
```

Comment:
```suggestion
[Product management](https://en.wikipedia.org/wiki/Product_management) is a collection of product development and product marketing activities within a products or services organization. The purpose of these activities is to develop a product that meets market demand on the one hand and to manage the product throughout its life cycle on the other. At a large company like Apple, thousands of people work on this every day. The end goal is to make a product or service that matches the market so well that as many as possible can be sold. The activities and responsibilities are centralized because only one organization develops the product and determines what happens to it.
```

Reply:
I think it is more appropriate to not link the space